truss/httperr.go

```diff
@@ -0,0 +1,28 @@
+package truss
+
+import "net/http"
+
+// HTTPError encodes information in an error type for the
+// github.com/go-kit/kit/tranport/http.DefaultErrorEncoder to create a
+// meaningful http response
+func HTTPError(err error, statusCode int, headers http.Header) error {
```

Comment:
Since we don't want `truss` to be a lib, I think this HTTPError type and struct should be moved to generated code. I feel it makes sense for it to live in `name-service/svc/transport_http.go` or potentially in a new file or sub-package within `name-service/svc/`.